lib/puppet/provider/s3_bucket/v2.rb

```diff
@@ -44,8 +44,11 @@ def exists?
   end
 
   def create
-    Puppet.debug("Creating S3 Bucket #{name}")
-    s3_client.create_bucket({bucket: name})
+    Puppet.debug("Creating S3 Bucket #{name} in #{target_region}")
```

Comment:
@zleswomp Sorry, I included some commits that weren't supposed to be added. I'll branch out and create a new pull request.